src/components/MainPage/ProgressBar/index.tsx

```diff
@@ -31,13 +31,15 @@ const ProgressBar = () => {
 
   return (
     <S.Container>
-      <S.ProgressWrapper>{Progress.container}</S.ProgressWrapper>
-      <S.GaugeWrapper>
-        <div>
-          당신의 감정 {growthRest}개가 <br />더 궁금해요!
-        </div>
-        {Progress.bar[`${percent}`]}
-      </S.GaugeWrapper>
+      <S.ProgressWrapper>
+        {Progress.container}
+        <S.GaugeWrapper>
+          <div>
+            당신의 감정 {growthRest}개가 <br />더 궁금해요!
+          </div>
+          {Progress.bar[`${percent}`]}
```

Comment:
``${}``이 없어도 될 것 같네요!
```suggestion
          {Progress.bar[percent]}
```
그리고, Progress 가 PascalCase로 되어있는데, 객체면, camelCase가 더 적합할 것 같아요!

Reply:
감사해요 반영했습니다! 스타일만 보다가 코드는 살펴볼 생각을 안했나봐요 😅
확인해보니 Progrss는 객체보다 컴포넌트로 분류된다고 생각돼서 케이싱 + 사용 방식을 헷갈리지 않게 수정했어요.